robotoff/models.py

```diff
@@ -69,10 +120,16 @@ class ProductInsight(BaseModel):
         "one of 'off', 'obf', 'opff', 'opf'",
         index=True,
     )
+
+    # This field refers to the number of unique IPs that have scanned this product.
     unique_scans_n = peewee.IntegerField(default=0, index=True)
+
+    # A reserved barcode is a barcode that is a 'per-weight' barcode.
+    # NOTE: currently this field is being set for 2*** barcodes, but is not read by anything.
```

Comment:
This field is used in the API to filter products with reserved barcodes: https://github.com/openfoodfacts/robotoff/blob/fce2c70308a75c8b8eef406ff5951fdf7735423b/robotoff/app/api.py#L852